.goreleaser.yml

```diff
@@ -308,4 +348,37 @@ brews:
     homepage: https://docs.confluent.io/confluent-cli/current/overview.html
     description: CLI for Confluent Cloud and Confluent Platform
     skip_upload: "{{ .Env.DRY_RUN }}"
-    test: system "#{bin}/confluent version"
\ No newline at end of file
+    test: system "#{bin}/confluent version"
+
+nfpms:
```

Comment:
I assume there's some additional configuration to send this off to packages.confluent.io?

Reply:
Context for anyone else reading: we ended up talking about this in a meeting and we'll probably handle this w/ blobs.